src/veh_interact.cpp

```diff
@@ -814,6 +834,29 @@ void veh_interact::do_install()
     display_name();
 }
 
+void veh_interact::redraw() const
+{
+    redrawwin(w_grid);
+    redrawwin(w_mode);
+    redrawwin(w_msg);
+    redrawwin(w_disp);
+    redrawwin(w_parts);
+    redrawwin(w_stats);
+    redrawwin(w_list);
+    redrawwin(w_details);
+    redrawwin(w_name);
```

Comment:
Is the `redrawwin` really required? No other part of the program calls, that's what's confusing me.

Or should it be called in other places as well? Maybe that can solve issues like #9835, #8514, as it marks the whole window as modified, forcing curses to refresh all of it.